### PR TITLE
Fix several devtools::check() issues

### PR DESCRIPTION
**Pull request by billdenney**
See one note below for one item where I'm unsure if the fix is correct.

### CODE REVIEW

File: R/monolixReadData.R [hunk 1]
```diff
@@ -406,7 +406,7 @@ rxUiGet.monolixCovarianceEstimatesSA <- function(x, ...) {
 #' @export
 rxUiGet.monolixCovariance <- function(x, ...) {
   .cov <- rxUiGet.monolixCovarianceEstimatesSA(x, ...)
-  .split <- ui$getSplitMuModel
+  .split <- x$getSplitMuModel
```
Comment (billdenney) [inline R/monolixReadData.R:409]: I think that this is the correct fix, but please confirm @mattfidler

Comment (mattfidler) [reply R/monolixReadData.R:409]: Yes this is correct @billdenney

**Comment by mattfidler** [comment]
We also need to include the following in the description:

```
While not required, you can get/install the R 'lixoftConnectors'
    package in the 'Monolix' installation, as described at the following url
    <https://monolix.lixoft.com/monolix-api/lixoftconnectors_installation/>.
    When 'lixoftConnectors' is available, R can 'Monolix' directly instead of
    using a command line.
```

And use singe quotes for software.  This is so that it can be accepted into CRAN eventually.

**Comment by mattfidler** [comment]
According to CRAN policies, you need to list how to get the R packages that are not available on CRAN in the description file.

This is what I did for `ggPMX`:

https://cran.rstudio.com/web/packages/ggPMX/index.html

You can see the `lixoftConnectors` is not linked

**Comment by billdenney** [comment]
That makes sense.  I added it back.

**Pull request merged**